<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labBase">
       <property name="text">
        <string>base:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBase">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labExp">
       <property name="text">
        <string>exponent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinExp">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labMod">
       <property name="text">
        <string>modulos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinMod">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="resultTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
